velox/exec/Driver.cpp

```diff
@@ -539,6 +539,11 @@ StopReason Driver::runInternal(
             curOperatorId_,
             kOpMethodIsBlocked);
         if (blockingReason_ != BlockingReason::kNotBlocked) {
+          VELOX_CHECK(
+              future.valid(),
+              "The operator {} is blocked but blocking future is not set by "
+              "isBlocked method.",
+              op->operatorType());
```

Comment:
There are more instances in runInternal() where we get blocking reason and a future from an operator.
For example, on line 557:
  blockingReason_ = nextOp->isBlocked(&future),

Should we add similar checks in all such code sites?

Reply:
Yes, fixed.